Sources/ArgumentParser/Parsable Properties/ArgumentHelp.swift

```diff
@@ -24,26 +35,31 @@ public struct ArgumentHelp {
   ///   flags don't include a value.
   public var valueName: String?
   
-  /// A Boolean value indicating whether this argument should be shown in
+  /// A visibility level indicating whether this argument should be shown in
   /// the extended help display.
-  public var shouldDisplay: Bool = true
```

Comment:
I dont think we can eliminate this property since it is part of our public API, but it can be deprecated and forward get/set to the new visibility property you've defined

Same for the initializer below

Reply:
Indeed, my bad